docs/UserGuide.md

```diff
@@ -202,13 +216,27 @@ Format: `add t/TITLE m/MILESTONE by/DEADLINE [d/DESCRIPTION] [c/CONTACTS]`
 * `MILESTONE` is the milestone tagged to the deliverable.
 * `MILESTONE` takes in numerical values separated by periods, e.g. `1.3`, `14.2.1`.
 * `DEADLINE` is the due date time of the deliverable in DD-MM-YYYY HH:mm format.
+* `DEADLINE` can be in the past but must not be earlier than the year 2019.
```

Comment:
Did we edit the code to validate this when we parse the datetime?

Reply:
yes I did in the other PR